closed/OpenJ9.gmk

```diff
@@ -205,7 +205,12 @@ define openj9_copy_tree_impl
   @$(TOUCH) $1/$(OPENJ9_MARKER_FILE)
 endef
 
-$(eval $(call generated_target_rules_build,build_jdk,$(BUILD_JDK)))
+# Temporary fix until we work out how to handle the build_jdk for cross compiles
+ifeq (1,$(OPENJDK_DOCKER_CC_ARM))
```

Comment:
Where does OPENJDK_DOCKER_CC_ARM come from?

Reply:
It's one of a set of env vars exported by the docker image I created for hosting arm cross compilation. ( https://github.com/eclipse/openj9/pull/622 )